messaging/propagate.go

```diff
@@ -73,7 +74,7 @@ type propagationContext interface {
 func NewPropagationFunc(c propagationContext, name string, f PropagationStore) (PropagationFunc, error) {
 	pid, err := c.ProtocolRegister(name, func(n *onet.TreeNodeInstance) (onet.ProtocolInstance, error) {
 		p := &Propagate{
-			sd:               &PropagateSendData{[]byte{}, 1000},
+			sd:               &PropagateSendData{[]byte{}, 100000},
```

Comment:
The timeout can be chosen by the caller, but it's not available upon creation of the protocol :( See https://github.com/dedis/cothority/pull/868/files/3d0b2cf919271a2456e5574e87d891cb831e9713#diff-1de3dc5ac1cd86438180ef0429247212R147

So it's variable, but we might have to wait to know what the user wants.